api/libproject/devstart.rb

```diff
@@ -80,9 +80,11 @@ def start_local_db_service()
   bm = Benchmark.measure {
     common.run_inline %W{docker-compose up -d db}
 
+    root_pass = Workbench.read_vars_file("db/vars.env")["MYSQL_ROOT_PASSWORD"]
+
     common.status "waiting up to #{deadlineSec}s for mysql service to start..."
     start = Time.now
-    until (common.run %W{docker-compose exec -T db mysqladmin ping --silent}).success?
+    until (common.run "docker-compose exec -T db mysql -p#{root_pass} --silent -e 'SELECT 1;'").success?
```

Comment:
For some reason `mysqladmin ping` seemingly was returning success before the database was actually ready. This resulted in:

```
ERROR 2013 (HY000): Lost connection to MySQL server at 'handshake: reading initial communication packet', system error: 11
```

on the first run after clearing your DB. Issuing a no-op SQL command SELECT 1 seems to work more reliably